ptbr/training_cli.py

```
@@ -0,0 +1,1038 @@
+def _resolve_data_path(path_value: str, config_dir: Path) -> Path:
+    """Resolve dataset paths relative to the YAML config directory."""
+    path = Path(path_value).expanduser()
+    if not path.is_absolute():
+        path = config_dir / path
+    return path
+
+
+def _launch_training(
+    cfg: dict,
+    output_folder: Path,
+    resume: bool,
```

Ruff (0.15.1)

[warning] Unused function argument: `resume`

(ARG001)